content/documents/dev/the_art_of_jamming_all_parts.html

```diff
@@ -185,7 +198,7 @@ <h4>The Main Rule</h4>
 
 <p>As described briefly in the explanation of the <code>MkObjectDirs</code> rule, this variable defines where targets should be located, i.e. where they are created. In this case it is set to be &quot;obj.&quot; with the platform appended. The variable <code>$(OSPLAT)</code> is one of the few variables actually compiled into Jam (and therefore not set in Jambase), and it is an all capital description of the CPU type (such as X86, PPC or SPARC.)</p>
 
-<p>Jam runs on many platforms, and a properly written Jamfile should be able to work on many of them, unmodified. For example, the Jamfile-engine should theoretically run on both x86 and PowerPC versions of BeOS as it is today (the term theoretically is used here since no one has tested the Jamfile-engine on a PowerPC machine yet.)</p>
+<p>Jam runs on many platforms, and a properly written Jamfile should be able to work on many of them, unmodified. For example, the Jamfile-engine should theoretically run any Haiku plattforms as it is today (the term theoretically is used here since no one has tested the Jamfile-engine on a non-x86 machine yet.)</p>
```

Comment:
```suggestion
<p>Jam runs on many platforms, and a properly written Jamfile should be able to work on many of them, unmodified. For example, the Jamfile-engine should theoretically run any Haiku platforms as it is today (the term theoretically is used here since no one has tested the Jamfile-engine on a non-x86 machine yet.)</p>
```